components/ingest-service/config/config_mgmt.go

```diff
@@ -114,84 +121,66 @@ func defaultConfig() aggregateConfig {
 				Running:   true,
 			},
 		},
+		ProjectUpdateConfig: ProjectUpdateConfig{
+			State: "not_running",
```

Comment:
I think I saw this elsewhere. Worth pulling into a constant? 🤔

Reply:
Thank you for pointing that out. I think I was being lazy. I updated them all in this commit. 
https://github.com/chef/automate/pull/132/commits/055ec162a0155d9212f6a737d328b13674e849f4